### PR TITLE
Update information-for-dns-records.md

### PR DESCRIPTION
**Pull request by msbemba**
added a step to remove the TXT or MX after verifying the domain in o365.

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: Office365-Admin/get-help-with-domains/information-for-dns-records.md [hunk 1]
```diff
@@ -51,6 +51,8 @@ description: "Learn to find the values/information you need to create DNS record
 5. Go to [Create DNS records at any DNS hosting provider](create-dns-records-at-any-dns-hosting-provider.md), and select your DNS host from the list of registrars to see the step-by-step instructions.
     
 6. Follow the steps for creating the TXT record (or MX record) at your DNS host, then verify the domain back in Office 365.
+
+7. The TXT record (or MX record) can be removed from your DNS host once the domain is verified in Office365.
```
Comment (JohanFreelancer9) [inline Office365-Admin/get-help-with-domains/information-for-dns-records.md]:
```suggestion
7. The TXT record (or MX record) can be removed from your DNS host once the domain is verified in Office 365.
```

**Comment by mypil** [comment]
@msbemba  - please merge the commit on the copy/edit review made by JohanFreelancer9 / nenonix  or make the necessary changes on the PR following the suggestions made. Thank you.

**Comment by msbemba** [comment]
@mypil Merged the Commits.

**Comment by mypil** [comment]
@pebaum - Can you please assist to check, approve and merge the content of this PR based on issue #493?

Thank you.

**Pull request merged**